lib/services/local/desktops.go

```diff
@@ -224,11 +288,6 @@ func (s *WindowsDesktopService) ListWindowsDesktops(ctx context.Context, req typ
 		return nil, trace.Wrap(err)
 	}
 
-	// If both HostID and Name are set in the filter only one desktop should be expected
-	if req.HostID != "" && req.Name != "" && len(desktops) == 0 {
-		return nil, trace.NotFound("windows desktop \"%s/%s\" doesn't exist", req.HostID, req.Name)
-	}
-
```

Comment:
Have you checked around to see if we're depending on this for something?

Reply:
Yeah, I didn't spot anything.

`ListWindowsDesktops` was added to support the unified resources view. It wasn't around when all the original desktop code was added, so I'm not surprised that nothing in the desktop code depends on it.